src/Illuminate/Console/Scheduling/ScheduleRunCommand.php

```diff
@@ -67,4 +105,5 @@ public function fire()
             $this->info('No scheduled commands are ready to run.');
         }
     }
+
```

Comment:
Fixed in 40ed862fc9b4243e33c8dbaee40c3e82e542c128. This was force push integrating changes requested by style ci.